### PR TITLE
Convert the REUSE config from dep5 to toml

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

dep5 has been deprecated and toml allows to define what to do if the copyright is in both the config and the file. The headers having broken header in the po files, this is a needed feature.

## Test coverage
- No tests: tooling change

- [X] **DONE**

## Links

Issue(s): #

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: REUSE.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# SPDX-License-Identifier: Apache-2.0
+version = 1
+SPDX-PackageName = "uyuni-tools"
+SPDX-PackageSupplier = "Uyuni Project <>"
+SPDX-PackageDownloadLocation = "https://github.com/uyuni-project/uyuni-tools"
+
+[[annotations]]
+path = ["mgradm/shared/ssl/testdata/**", "go.mod", "go.sum", "uyuni-tools.changes*", "uyuni-tools.spec", ".tito/**"]
+precedence = "aggregate"
+SPDX-FileCopyrightText = "2023 SUSE LLC"
```
[the remark this answers is not in the document]
Comment (deneb-alpha) [reply REUSE.toml:12]: looking at https://spdx.github.io/spdx-spec/v2.2.2/file-information/ and https://spdx.github.io/spdx-spec/v2.2.2/file-tags/it seems we can use also the range of dates

[the remark this answers is not in the document]
Comment (cbosdo) [reply REUSE.toml]: Fixed! thanks for the heads up

File: REUSE.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2023 SUSE LLC
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply REUSE.toml]: Used a range there

**Review by deneb-alpha — APPROVED** [review]
LGTM, thanks!

**Pull request merged**